brackend/tasks/tasks.py

```diff
@@ -48,19 +50,38 @@ def save_new_user(username, firebase_id):
 
 
 def get_user_ids():
+    """
+    returns a list of all users in the database
+    """
     engine = EngineGetter.get_or_create_engine()
     with Session(engine) as session:
         users = session.query(User).all()
         return [u.id for u in users]
 
 
 def get_tournament_ids():
+    """
+    returns a list of all tournaments in the database
+    """
     engine = EngineGetter.get_or_create_engine()
     with Session(engine) as session:
         users = session.query(Tournament).all()
         return [u.id for u in users]
 
 
+def delete_tournament(tid):
+    """
+    delete tournament from the database
+    """
+    engine = EngineGetter.get_or_create_engine()
+    with Session(engine) as session:
+        tournament = session.query(Tournament).where(Tournament.id == tid)[0]
```

Comment:
I think we should do tournament ownership check here. If you add a user_id param to this function, you could do an owner check by querying for a `UserTournament` where `UserTournament.role == UserRole.organizer` and  ` UserTournament.user_id == user_id`. 

(You can get the user_id in the endpoint's file by doing `g.user.id`)

Reply:
cool, i think i've got one up now, give it a look